app/Order.php

```diff
@@ -54,6 +54,28 @@ public function user()
     }
 
     /**
+     * Update Total Price of Order
+     *
+     * @param int $id It is id of order need updateTotalPrice.
+     *
+     * @return bool
+     */
+    public function updateTotalPrice($id)
```

Comment:
Hàm này viết đúng rồi, nhưng không cần truyền `$id` vào làm gì, Eloquent của Laravel viết theo dạng ActiveRecord nên mỗi object của Order có property là các column trong DB luôn. Nên sẽ lấy được id từ ngay trong chính object đó. 
`$this->id` , không cần phải query lại.